numbers/orm.tsv

```diff
@@ -1,90 +1,90 @@
-0	duwwa
+0	duwwaa
 1	tokko
 2	lama
-3	sadii
```

Comment:
`sadii`

Reply:
It's `sadi`, not `sadii`
The i sound is not elongated unless you're making some kind of melody, in which case you can do same for other numbers by elongating the sound of last vowel.

Reply:
Thank you for your review. 

Actually in the formal writing, 3 is written as `sadi`  (Check elementary Text books in Afaan Oromoo), except in some contexts where `sadi` takes different form such as `sadan`, 330 (`Dhibba sadiifi  soddoma`).  In this dataset, we're referring to 3 alone, not in context like `Kitaaba sadan sana.` or in numbers like 330. 
For example this is from Gadaa grade 8 text book.

![image](https://user-images.githubusercontent.com/40395348/214786960-6c0836c0-4f45-4652-96cb-bb339a993025.png)

I know there are different dialect in Oromo language, maybe that's why it's written `sadii` in bible. But personally I have never seen 3 written as `sadii` before in formal writing.  The person who wrote the initial commit is also using different dialect, maybe? I have never seen 10 written as `Khudan`.  I guess it's typo. 

**In such dialect differences, I believe we should use the formal writing used in school text books.**  That's what we have been learning across Oromia.

Reply:
I guess we should wait for a professional to come. I asked my sources (close family, which are from West and Center) and they know it as `sadii`, it could be the dialect as you said. 

I could also send for more reviewers but don't know if it's allowed. [kylebgorman](https://github.com/kylebgorman) will get more formal reviewers

Reply:
I find the evidence from the text book to be pretty probative so my inclination would be to use _sadi_.